extensions/guacamole-auth-sso/modules/guacamole-auth-sso-openid/src/main/java/org/apache/guacamole/auth/openid/conf/ConfigurationService.java

```diff
@@ -126,6 +128,12 @@ public class ConfigurationService {
 
     };
 
+    private static final StringGuacamoleProperty OPENID_ATTRIBUTES_CLAIM_TYPE =
```

Comment:
All newly-added functions, member variables, etc. need to be documented with JavaDoc comments (see surrounding code).